src/lib/src/hero/hero.component.css

```diff
@@ -0,0 +1,146 @@
+/**
+ * Banner text
+ */
+.banner {
+  background: #818181 url('ngx-prx-styleguide/assets/images/towers.png') repeat;
```

Comment:
Would you mind throwing these 3 pngs through a compressor before we merge this?  I seem to recall that beta did it's own image compressing, so these were fairly un-optimized.